hledger, hledger-ui, hledger-web: update/add (#12824)
tldr-pages/tldr: pages/common/hledger-web.md

# hledger-web

> > A robust, friendly plain text accounting app (the web UI & API).

Reviewer: ```suggestion
> Web interface and API to `hledger`.
> A robust, friendly plain text accounting app.
```
Maybe, we can separate the descriptions to highlight the web UI and API part from the generic description.

> More information: <https://hledger.org>.

- Start the web app, and a browser if possible, for local viewing and adding:

`hledger-web`

- As above but with a specified file, and allowing editing of existing data:

> `hledger-web --file {{path/to/file.journal}} --allow edit`

Author: Same question about losing realism of example filenames, as above..

> `hledger-web --file {{path/to/file.journal}} --allow edit`

Reviewer: > Same question about losing realism of example filenames, as above..

Updated the suggestion.

> `hledger-web --file {{path/to/file.journal}} --allow edit`

Author: In this case, 2024-prices.journal is too specific and not quite what a user would normally do. :) I'll fix.

# hledger-web

> Web interface and API to `hledger`.
> A robust, friendly plain text accounting app.
> More information: <https://hledger.org/hledger-web.html>.

- Start the web app, and a browser if possible, for local viewing and adding only:

`hledger-web`

- As above but with a specified file, and allow editing of existing data:

`hledger-web --file {{path/to/file.journal}} --allow edit`

- Start just the web app, and accept incoming connections to the specified host and port:

`hledger-web --serve --host {{my.host.name}} --port 8000`

- Start just the web app's JSON API, and allow only read access:

`hledger-web --serve-api --host {{my.host.name}} --allow view`

- Show amounts converted to current market value in your base currency when known:

`hledger-web --value now --infer-market-prices`

- Show the manual in Info format if possible:

`hledger-web --info`

> - Show options:

Reviewer: ```suggestion
- Display help:
```
